utilities_common/netstat.py

```diff
@@ -1,10 +1,19 @@
 # network statistics utility functions #
 
 import json
+from collections import namedtuple
 
 STATUS_NA = 'N/A'
 PORT_RATE = 40
 
+rates_key_list = [ 'RX_BPS', 'RX_PPS', 'RX_UTIL', 'TX_BPS', 'TX_PPS', 'TX_UTIL' ]
```

Comment:
I don't think these should be declared in this file. This needs to go inside portstat and rifstat respectively. This file is meant to be like a library for various formatting APIs